assignments/DOM Basics/02/index.html

```diff
@@ -47,6 +47,32 @@ <h2>Query Selector All</h2>
     <div class="demo-query-all">Access me by query all (2)</div>
     <script>
       // your code goes here
+      document.getElementById("demo").style.border = "1px solid purple";
+      // document.querySelectorAll(".demo").style.border = "1px solid orange";------ why not..?
+      
+      var orange = document.getElementsByClassName("demo");
+      var i = 0;
+      while (i<orange.length){
```

Comment:
Take care of spacing and indentations. 

Go through and refactor your code https://javascript.info/structure